code/script.js

```diff
@@ -255,71 +273,180 @@ const checkQuestion = () => {
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
-  if (category === 'hair' || category === 'eyes') {
+  if (category === 'Skin Color' || category === 'Skin/body texture') {
+    if (category === 'Skin Color') {
+      if (secret.skinColor === value) {
+        filterCharacters(true)
+      }
+      else {
+        filterCharacters(false)
+      }
+    }
+    else {
+      if (secret.skinTexture === value) {
+        filterCharacters(true)
+      }
+      else {
+        filterCharacters(false)
+      }
+    }
 
-  } else if (category === 'accessories' || category === 'other') {
+  } else if (category === 'Attributes' || category === 'Other attributes') {
+
+    if (category === 'Attributes') {
+      if (secret.attributes.includes(value)) {
+        filterCharacters(true)
+      }
+      else {
+        filterCharacters(false)
+      }
+    }
+    else {
+      if (secret.other.includes(value)) {
+        filterCharacters(true)
+      }
+      else {
+        filterCharacters(false)
+      }
+    }
 
   }
 }
 
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
   const { category, value } = currentQuestion
+  let stringPropertyName = ''  //Mappa (translate) Skin color och Skin texture
+  let arrayPropertyName = '' //Mappa (translate) Attribute and Other attributes
+
+
+
+
   // Show the correct alert message for different categories
-  if (category === 'accessories') {
+  if (category === 'Skin Color') {
+    stringPropertyName = 'skinColor';
     if (keep) {
       alert(
-        `Yes, the person wears ${value}! Keep all people that wears ${value}`
+        `Yes, the dinosaur have ${value} skin! Keep all dinosaurs that has ${value} skin`
       )
     } else {
       alert(
-        `No, the person doesn't wear ${value}! Remove all people that wears ${value}`
+        `No, the dinosaur doesn't have ${value} skin! Remove all dinosaurs that has ${value} skin`
       )
     }
-  } else if (category === 'other') {
-    // Similar to the one above
-  } else {
+
+  }
+  else if (category === 'Skin/body texture') {
+    stringPropertyName = 'skinTexture'
     if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all people with yellow hair"
+      alert(
+        `Yes, the dinosaur have ${value}! Keep all dinosaurs that has ${value}`
+      )
     } else {
-      // alert popup that says something like: "No, the person doesnt have yellow hair! Remove all people with yellow hair"
+      alert(
+        `No, the dinosaur doesn't have ${value}! Remove all dinosaurs that has ${value}`
+      )
+    }
+
+  } else if (category === 'Attributes') {
+    arrayPropertyName = 'attributes';
+    if (keep) {
+      alert(
+        `Yes, the dinosaur ${value}! Keep all dinosaurs that has ${value}`
```

Comment:
Consider changing the alert message when guessing on attributes. For example, deleting the word "has" makes it more suitable.

Reply:
Thank you for this input! I had so much trouble with the answer on the modal message. Because some of the words has different endings.